rex/resource.py

```diff
@@ -293,13 +381,20 @@ def _extract_multi_list_slice(self, ds_slice, list_len):
         """
         zip_slices = self._make_list_slices(ds_slice, list_len)
 
-        out = None
+        out_shape = self._get_out_arr_shape(ds_slice)
+
+        out = np.zeros(out_shape, dtype=self.dtype)
+        start = 0
+        print(out.shape)
         for s in zip(*zip_slices):
-            arr = np.expand_dims(self._ds[s], axis=0)
-            if out is None:
-                out = arr
-            else:
-                out = np.append(out, arr, axis=0)
+            print(s)
+            arr_slice, stop = self._get_out_arr_slice(s, start)
+            print(arr_slice)
```

Comment:
Print statements?

Reply:
Did I not catch all of them. I was debuging

Reply:
fixed

Reply:
looks good!